### PR TITLE
Fix "flowing" liquids not being included

### PR DESCRIPTION
**Pull request by Flonja**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -475,13 +475,13 @@ function Physics (mcData, world) {
     }
   }
 
-  function isMaterialInBB (world, queryBB, type) {
+  function isMaterialInBB (world, queryBB, ...types) {
```
Comment (rom1504) [inline index.js:478]: why the ... ?

Comment (Flonja) [reply index.js:478]: To be able to handle 1 or multiple types

Comment (rom1504) [reply index.js:478]: that does not work

if you give to a ...types a list, the argument will be a list of list

Comment (rom1504) [reply index.js:478]: Can you remove the ... and change all calls to pass a list ? It's simpler and better

**Pull request merged**